assets/js/modules/analytics-4/utils/account.js

```diff
@@ -62,5 +64,6 @@ export function getAccountDefaults(
 		timezone: countryCodesByTimezone[ timezone ]
 			? timezone
 			: _fallbackTimezone,
+		[ ENHANCED_MEASUREMENT_ENABLED ]: true,
```

Comment:
This will set the default value when the account creation form state is initialized with the rest here.